docs/test-suite/installation-configuration.md

```diff
@@ -8,10 +8,15 @@ This page covers the requirements, installation and configuration of the [EESSI
 
 The EESSI test suite requires 
 
-* Python >= 3.6 
+* Python >= 3.7
 * [ReFrame](https://reframe-hpc.readthedocs.io) v4.3.3 (or newer)
 * [ReFrame test library (`hpctestlib`)](https://reframe-hpc.readthedocs.io/en/stable/hpctestlib.html)
 
+??? note "(If your system Python version is <= 3.6, click here for some tips)
+
+    * You can upgrade the python version in a virtual environment. [Link](https://docs.python.org/3/library/venv.html)
```

Comment:
Wait, what? You cannot install a newer python in a virtualenv, right?

You can (according to my AI friend) do this with `pyenv` https://github.com/pyenv/pyenv. I think that's also the tool that Sam referred to. As far as I understand, `pyenv` installs into your homedir by default (i.e. perfect for userspace installs) and allows you to install new python versions with `pyenv install <version>; pyenv global <version>  # Sets tje default python version to use for your user; pyenv --version`. Note: I haven't tried myself, could be good to give it ago before you put it in the docs to be sure this works. I'd probably just say: "Install pyenv by following their documentation https://github.com/pyenv/pyenv", and then give the explicit instruction on how to install a different python version.

Reply:
Yeah, true. I read it wrong in the stackoverflow answer and in the documentation as well. :D The documentation states that one can upgrade the packages in the venv to a newer version as long as a newer python version is in place.
Indeed what Sam was talking was about `pyenv`.
This is only possible with a local package manager such as `conda`. Not sure if we want to recommend `conda` here.

Reply:
By "This" I meant upgrading the python version. It is possible using
1. `pyenv` indeed does not require existing python versions, it is just a python version manager. I just tested it and it works.
2. `conda` is a second alternative but I was not sure whether we want to pitch conda. :)